cves/CVE-2013-1664.yml

```diff
@@ -1,296 +1,314 @@
-CWE:
-CWE_note:
-nickname_instructions: |
-  A catchy name for this vulnerability that would draw attention it. If the
-  report mentions a nickname, use that. Must be under 30 characters.
-  Optional.
-nickname:
+CVE: CVE-2013-1664
+yaml_instructions: |
+  ===YAML Primer===
+  This is a dictionary data structure, akin to JSON.
+  Everything before a colon is a key, and the values here are usually strings
+  For one-line strings, you can just use quotes after the colon
+  For multi-line strings, as we do for our instructions, you put a | and then
+  indent by two spaces
+
+  For readability, we hard-wrap multi-line strings at 80 characters. This is
+  not absolutely required, but appreciated.
+curated_instructions: |
+  If you are manually editing this file, then you are "curating" it. Set the
+  entry below to "true" as soon as you start. This will enable additional
+  integrity checks on this file to make sure you fill everything out properly.
+  If you are a student, we cannot accept your work as finished unless curated is
+  set to true.
+curated: true
+reported_instructions: |
+  What date was the vulnerability reported to the security team? Look at the
+  security bulletins and bug reports. It is not necessarily the same day that the
+  CVE was created.  Leave blank if no date is given.
+  Please enter your date in YYYY-MM-DD format.
+reported_date: 2013-02-13
+announced_instructions: |
+  Was there a date that this vulnerability was announced to the world? You can
+  find this in changelogs, blogs, bug reports, or perhaps the CVE date. A good
+  source for this is Chrome's Stable Release Channel
+  (https://chromereleases.googleblog.com/).
+  Please enter your date in YYYY-MM-DD format.
+announced_date: 
+published_instructions: |
+  Is there a published fix or patch date for this vulnerability?
+  Please enter your date in YYYY-MM-DD format.
+published_date: 2013-04-02
+description_instructions: |
+  You can get an initial description from the CVE entry on cve.mitre.org. These
+  descriptions are a fine start, but they can be kind of jargony.
+
+  Rewrite this description IN YOUR OWN WORDS. Make it interesting and easy to
+  read to anyone with some programming experience. We can always pull up the NVD
+  description later to get more technical.
+
+  Try to still be specific in your description, but remove project-specific
+  stuff. Remove references to versions, specific filenames, and other jargon
+  that outsiders to this project would not understand. Technology like "regular
+  expressions" is fine, and security phrases like "invalid write" are fine to
+  keep too.
+
+  Your target audience is people just like you before you took any course in
+  security
+description: |
+  XML libraries for Python versions 3.4, 3.3, 3.2, 3.1, 2.7 and 2.6 allowed 
+  hostile attackers to make use of Denial-of-Service or DOS vulnerabilites via
+  an XML Entity Expansion or XEE attack. This attack was used against various

```

Comment:
It would be great if you could explain a bit more about what is XML entity expansion or XEE attack.

Reply:
Also possibly specifically what makes them hostile would also be intersting.

Reply:
Ok. I will add that